sklearn/neural_network/tests/test_mlp.py

```diff
@@ -27,8 +27,7 @@
                                    assert_equal, ignore_warnings)
 from sklearn.utils.testing import assert_raise_message
 
-
-np.seterr(all='warn')
+# np.seterr(all='warn')
```

Comment:
please remove them entirely then, not just uncomment ;)

Reply:
@NicolasHug Commented them to just test the changes. Removed them now